docker/README.md

```diff
@@ -0,0 +1,79 @@
+# Using Docker to run garage
+
+Currently there are three types of garage images:
+  - ci: used for the continuous integration of garage in TravisCI and not
+    intended for final users.
+  - headless: garage without environment visualization.
+  - NVIDIA: garage with environment visualization using an NVIDIA graphics
+    card.
+
+
+## Headless image
+
+To run an example launcher in the container, execute:
+```
+docker run -it --rm rlworkgroup/garage-headless python examples/tf/trpo_cartpole.py
+```
+
+This image does not come with a MuJoCo key, so if you want to run environments
+using MuJoCo, the environment variable `MJKEY` has to be set with the
+corresponding content of the key file in your file system:
+```
+# New line characters in MuJoCo key files (they're in DOS format) are
+# replaced by '\n' to pass the file as a command argument.
+export MJKEY="$(sed ':a;N;$!ba;s/\r\r//g;s/\n/\\n/g' <your_key_path>)"
```

Comment:
Do not instruct users to export their MJKEY. This is a security risk.

It's extremely easy to include your MJKEY as an argument

MJKEY="$(cat ~/.mujoco/mjkey.txt)"

Reply:
I agree that is safer to pass the MJKEY file as a variable to the makefile, but the cat command is not enough because once MJKEY is expanded in the recipe, its first return/newline will indicate the end of the build command, and the rest of the build command will remain unprocessed, generating an error.
For example:
```
(garage) aigonzal@gonzaiva:~/ivanWorkspace/garage$ make build-garage-headless MJKEY="$(cat ~/.mujoco/mjkey.txt)"
docker build \
	-f docker/Dockerfile \
	--build-arg MJKEY="MuJoCo Pro Individual license activation key, number ????, type ?.
/bin/sh: 3: Syntax error: Unterminated quoted string
Makefile:14: recipe for target 'build-garage-headless' failed
make: *** [build-garage-headless] Error 2
```
That's why I'm using sed.